### PR TITLE
chore: update readme

### PR DESCRIPTION
**Pull request by kristianjokela**
## Description
Some small updates to the docs that hopefully clarify the usage of release-please and a note about the importance of keeping the dependencies in sync.
### Closes

**[DEV-XXX](https://helsinkisolutionoffice.atlassian.net/browse/DEV-XXX):**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -344,6 +349,10 @@ And all docs are available here: [release-please docs](https://github.com/google
 
 Use [Conventional Commits](https://www.conventionalcommits.org/) to ensure that the changelogs are generated correctly.
 
+### Releasable units
+
+Release please goes through commits and tries to find "releasable units" using commit messages as guidance - it will then add these units to their respective release PR's and figures out the version number from the types: `fix` for patch, `feat` for minor, `feat!` for major. All the other types will not be included in the changelog. So, you can use for example `chore` or `refactor` to do work that does not need to be included in the changelog and won't bump the version.
```
Comment (karisal-anders) [inline README.md:354]: How would you feel about this change?
"All the other types will not be included in the changelog." → "None of the other types will be included in the changelog unless they are [language specific releaseable units](https://github.com/googleapis/release-please#release-please-bot-does-not-create-a-release-pr-why)". Or if you don't want to go into the language specific releaseable units then just "All the other types will not be included in the changelog." → "None of the other types will be included in the changelog."?

File: README.md [hunk 2]
```diff
@@ -336,6 +339,8 @@ Each time you merge a "normal" pull request, the release-please-action will crea
 
 To create a new release for an app, this release PR is merged, which creates a new release with release notes and a new tag. This tag will be picked by Azure pipeline and trigger a new deployment to staging. From there, the release needs to be manually released to production.
 
+When merging release PRs, make sure to use the "Rebase and merge" (or "Squash and merge") option, so that Github doesnt create a merge commit. This is important, because the release-please-action does not work correctly with merge commits (there's an open issue you can track: [Chronological commit sorting means that merged PRs can be ignored ](https://github.com/googleapis/release-please/issues/1533)).
```
Comment (karisal-anders) [inline README.md:342]: typo: doesnt → doesn't

File: README.md [hunk 3]
```diff
@@ -357,14 +366,17 @@ When adding a new app, add it to both the [release-please-config.json](./release
 
 ### Troubleshoting release-please
 
-If you were expecting a new release PR to be created, but nothing happened, there's probably one of the older release PR's in pending state.
-
-First make sure there's no open release PR already; if there is, the work is now included on this one (this is the normal scenario).
+If you were expecting a new release PR to be created or old updated, but nothing happened, there's probably one of the older release PR's in pending state or action didn't run.
```
Comment (karisal-anders) [inline README.md:369]: How about "If you were expecting a new release PR to be created or old updated" → "If you were expecting a new release PR to be created or an old one to be updated"?

File: README.md (hunk 1 above)
Comment (kristianjokela) [reply README.md:354]: "None of the other types will be included in the changelog" might be better, so not to confuse people with the language spesific -things.

File: README.md [hunk 4]
```diff
@@ -344,6 +349,10 @@ And all docs are available here: [release-please docs](https://github.com/google
 
 Use [Conventional Commits](https://www.conventionalcommits.org/) to ensure that the changelogs are generated correctly.
 
+### Releasable units
+
+Release please goes through commits and tries to find "releasable units" using commit messages as guidance - it will then add these units to their respective release PR's and figures out the version number from the types: `fix` for patch, `feat` for minor, `feat!` for major. None of the other types will not be included in the changelog. So, you can use for example `chore` or `refactor` to do work that does not need to be included in the changelog and won't bump the version.
+
```
Comment (karisal-anders) [inline README.md:355]: Typo: "None of the other types will not be included in the changelog." → "None of the other types will be included in the changelog." i.e. not double negation.

**Review by karisal-anders — APPROVED** [review]

**Pull request merged**